app/views/candidate_interface/decisions/offer.html.erb

```diff
@@ -12,20 +12,22 @@
       <%= f.govuk_error_summary %>
 
       <h1 class="govuk-heading-xl">
+        <span class="govuk-caption-xl"><%= @application_choice.course.provider_and_name_code %></span>
```

Comment:
Sorry, can we drop this please? I think it was left in the prototype as a legacy, rather than something we deliberately added back in.
```suggestion
```

Reply:
Have done